src/Util.py

```diff
@@ -14,76 +14,60 @@ class that handles the interaction between the filesystem and the program's
-        =====================================================================
-        """
-        objects = []
-        loaded = [] # List of files we have already loaded
-        filestack = [filename] # Keep track of files we still need to load
-        while len(filestack) > 0:
-            # Get the next file to parse, skip if already done
-            fname = filestack[-1]
-            del filestack[-1]
-            if fname in loaded:
-                print("'{}' already loaded, skipping...".format(fname))
-                continue
-            
-            # Get the object and add to our object list
-            obj = Util.get_yaml_object(fname)
-            loaded.append( fname )
-            # If we include other files, add them to the stack
-            if 'include' in obj:
-                filestack.extend( obj['include'] )
-                del obj['include'] # Remove includes from our yaml object
-            objects.append( obj )
-        # TODO: Parse object list into *something*
-        # For now, just toss all objects into our _data list
-        Util._data.extend( objects )
+def get_yaml_object(filename : str) -> (bool, dict):
+    """
+    Wrapper method for yaml.safe_load(), loads an object from a yaml file
+    """
+    with open( filename, 'r' ) as stream:
+        data = yaml.safe_load( stream )
+        return data
 
-    @staticmethod
-    def get_yaml_object( filename : str ) -> (bool, dict):
-        """
-        Wrapper method for yaml.safe_load(), loads an object from a yaml file
-        """
-        with open( filename, 'r' ) as stream:
-            data = yaml.safe_load( stream )
-            return data
+def get_yaml_filename(directory : str, name_prefix : str):
+    yml_extension = path.join(directory, name_prefix + ".yml")
+    yaml_extension = path.join(directory, name_prefix + ".yaml")
+    if path.isfile(yml_extension):
+        return yml_extension
+    if path.isfile(yaml_extension):
+        return yaml_extension
+    raise GameError("Could not find yaml file '{}'.yaml".format(name_prefix))
 
-    
+
+class Util():
     @staticmethod
-    def clear_data() -> None:
-        """
-        =====================================================================
-        Clears the list of data this class holds
-        =====================================================================
-        Parameters:
-            None
-        =====================================================================
-        Returns:
-            None
-        =====================================================================
-        """
-        Util.data.clear()
\ No newline at end of file
+    def load_game(directory : str) -> Game:
+        enemies = load_data( get_yaml_filename("enemies") )
+        rooms = load_data( get_yaml_filename("rooms") )
+        items = load_data( get_yaml_filename("items") )
+        player = load_data( get_yaml_filename("player") )
+        return Game(enemies, items, rooms, player)
```

Comment:
This will need to be modified, since enemies is a list of data straight from the YAML, it should be converted into a list of `EnemyTemplate` objects that can be used to generate enemies in game.
Similar work will need to be done for the others as well.